src/translations/vnote_zh_CN.ts

```diff
@@ -3431,7 +3431,7 @@
     </message>
     <message>
         <location filename="../vmainwindow.cpp" line="1045"/>
-        <source>Custom some standard shortcuts</source>
+        <source>Customize some standard shortcuts</source>
```

Comment:
`.ts` file is generated by Qt according to other source files, so there is no need to update this. I will update it before a new release.

Reply:
Alright, I have no real experience in Qt so I didn't know this.